test/javascript/index.html

```diff
@@ -7,11 +7,12 @@
     <script src="../../node_modules/jquery/dist/jquery.js"></script>
     <script src="../../_build/assets/js/foundation.js"></script>
     <script src="lib/mocha.js"></script>
+    <link rel="stylesheet" href="lib/custom-mocha.css" />
     <link rel="stylesheet" href="lib/mocha.css" />
     <link rel="stylesheet" href="../../_build/assets/css/foundation-float.css">
     <link rel="stylesheet" href="../../node_modules/motion-ui/dist/motion-ui.css">
   </head>
-  <body>
+  <body id="test-body">
```

Comment:
I don't understand why this is in this PR. But as you said it, you can move this change to its own PR.